homeassistant/components/rachio/switch.py

```diff
@@ -141,6 +156,19 @@ def start_multiple(service: ServiceCall) -> None:
         else:
             raise HomeAssistantError("No matching zones found in given entity_ids")
 
+    platform = entity_platform.async_get_current_platform()
+    platform.async_register_entity_service(
+        SERVICE_START_WATERING,
+        {
+            vol.Optional(ATTR_DURATION): cv.positive_int,
```

Comment:
`ATTR_DURATION` is used here, but it looks like `KEY_DURATION_MINUTES` is used in `turn_on`?

Reply:
Yep, sure was